examples/gallery/images/grdgradient_shading.py

```diff
@@ -0,0 +1,25 @@
+import pygmt
+
+# Define region of interest around Caucasus 
+region = [39, 50, 40, 45]
+
+# Load sample grid (3 arc-seconds global relief) in target area
+grid = pygmt.datasets.load_earth_relief(resolution="03s", region=region)
+
+cmap = pygmt.makecpt(cmap='terra', series=[-5000, 5000])
+fig = pygmt.Figure()
+with fig.subplot(nrows=3, ncols=3, figsize=("20c", "20c")):
+    for i, rad in enumerate(['210/0','210/45', '210/90']):#, 0.5, 1]:
```

Comment:
```suggestion
    for i, rad in enumerate(['210/0','210/45', '210/90']):
```
That's not used anymore, right?

Reply:
I'm trying to fix this conflict. Thanks to your remind!